src/sentry/integrations/metric_alerts.py

```diff
@@ -169,7 +169,7 @@ def metric_alert_attachment_info(
             metric_value = get_metric_count_from_incident(incident_info)
 
     text = ""
-    if metric_value is not None:
```

Comment:
Should we only do this is `incident is None`?

Reply:
I believe this is obsolete per the other conversation